translate/cloud-client/translate_v3_list_glossary_test.py

```
@@ -34,7 +34,7 @@ def glossary():
     yield glossary_id
 
     try:
-        translate_v3_delete_glossary.sample_delete_glossary(PROJECT_ID, glossary_id)
+        translate_v3_delete_glossary.delete_glossary(PROJECT_ID, glossary_id)
     except Exception:
         pass
```

Can we update this to not swallow exceptions? This would have been caught otherwise.

---

Okay, I will make that change

---

Did you mean for all of other samples inside translate v3?

---

I mean short term for this specific sample, but if it's done in other samples they'll need to be adjusted sooner or later.

---

I will adjust then all here.